### PR TITLE
pat-464 rename cro column to crn in preparation for probation referral

### PR DESCRIPTION
**Pull request by lbennett-moj**

### CODE REVIEW

File: migrations/20200120145041_rename_nominal_cro_column_crn.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    }),
+    knex.schema.alterTable('nominal', table => {
+      table.string('crn').nullable()
+    }),
+  ])
+
+exports.down = knex =>
+  Promise.all([
+    knex.schema.table('nominal', table => {
+      table.dropColumn('crn')
```
Comment (timharrison-moj) [inline migrations/20200120145041_rename_nominal_cro_column_crn.js]: Should it recreate the cro column in the downgrade method to put it back?

File: server/services/offenderService.js [hunk 2]
```diff
@@ -137,7 +137,6 @@ module.exports = function createOffendersService(elite2ClientBuilder, offenderSe
     let response = {}
     if (!isNilOrEmpty(prisoners)) {
       const pnc = isNilOrEmpty(prisoners[0].pncNumber) ? '' : prisoners[0].pncNumber
-      const cro = isNilOrEmpty(prisoners[0].croNumber) ? '' : prisoners[0].croNumber
       const latestBookingId = isNilOrEmpty(prisoners[0].latestBookingId) ? '' : prisoners[0].latestBookingId
```
Comment (timharrison-moj) [inline server/services/offenderService.js]: Is the CRN number available in the elite2api - prisoner or booking data?

Comment (lbennett-moj) [reply server/services/offenderService.js]: no, I checked - we don't store it in nomis

Comment (lbennett-moj) [reply server/services/offenderService.js]: delius has the optional nomis Id to match to nomis records but not vice versa

File: migrations/20200120145041_rename_nominal_cro_column_crn.js (hunk 1 above)
Comment (lbennett-moj) [reply migrations/20200120145041_rename_nominal_cro_column_crn.js]: good point

**Pull request merged**